src/mcp_agent/mcp/mcp_aggregator.py

```
@@ -570,83 +809,139 @@ async def get_prompt(
+                            "progress_action": ProgressAction.SHUTDOWN,
+                            "server_name": server_name,
+                            "agent_name": self.agent_name,
+                        },
+                    )
+                    span.add_event(
+                        "temporary_connection_closed",
+                        {"server_name": server_name, "agent_name": self.agent_name},
+                    )
+
+            # Add namespaced name and source server to the result
+            # TODO: saqadri (FA1) - this code shouldn't be here.
```

Pylint (3.3.7)

[warning] TODO: saqadri (FA1) - this code shouldn't be here.

(W0511)